README.md

```diff
@@ -108,6 +108,26 @@ adb push tests/tests /data/local/tmp
 adb shell /data/local/tmp/tests
 ```
 
+## Using Docker
+
+The `Dockerfile` included in this repo will provision a Docker image suitable
+for building and running tests for both the Linux and Android platforms.
+
+```
+# Build Docker image from Dockerfile
+docker build -t "objectstore" .
+# Run bash interactively from the built Docker image,
+# mounting the current directory
+docker run -it -v `pwd`:`pwd` -w `pwd` objectstore bash
```

Comment:
I'd prefer something like `docker run --rm -it -v $(pwd):/tmp -w /tmp objectstore bash`